### PR TITLE
new blog post: membuat vite agar mendukung browser lama

### PR DESCRIPTION
**Pull request by bulukucing**
@elianiva

### CODE REVIEW

**Comment by bulukucing** [comment]
sorry, my bad. im not testing it yet

File: content/membuat-vite-agar-mendukung-browser-lama.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+---
+title: Membuat Vite Agar Mendukung Browser Lama
+desc: menambahkan plugin lagacy pada vite agar mendukung browser lama
```
Comment (aldy505) [inline content/membuat-vite-agar-mendukung-browser-lama.md:3]: typo: lagacy -> legacy

File: content/membuat-vite-agar-mendukung-browser-lama.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+Agar project yang kita buat bisa berjalan pada browser lama,
+
+kita perlu menambahkan `@vitejs/plugin-legacy` pada project kita,
+
+### Install @vitejs/plugin-legacy
+
+menggunakan npm:
+
+```sh
+npm install @vitejs/plugin-lagacy
```
Comment (aldy505) [inline content/membuat-vite-agar-mendukung-browser-lama.md:29]: typo. lagacy -> legacy.

File: content/membuat-vite-agar-mendukung-browser-lama.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+```
+
+atau jika menggunakan yarn:
+
+```sh
+yarn add @vitejs/plugin-lagacy
```
Comment (aldy505) [inline content/membuat-vite-agar-mendukung-browser-lama.md:35]: typo. lagacy -> legacy

File: content/membuat-vite-agar-mendukung-browser-lama.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+```
+
+### Konfigurasi plugin
+
+setelah berhasil menginstall `@vitejs/plugin-lagacy`
```
Comment (aldy505) [inline content/membuat-vite-agar-mendukung-browser-lama.md:40]: typo. lagacy -> legacy.

**Review by aldy505 — CHANGES_REQUESTED** [review]
typo

File: content/membuat-vite-agar-mendukung-browser-lama.md (hunk 3 above)
Comment (bulukucing) [reply content/membuat-vite-agar-mendukung-browser-lama.md]: fixed

**Comment by bulukucing** [comment]
fixed typo and update content

File: content/membuat-vite-agar-mendukung-browser-lama.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
+---
+title: Membuat Vite Agar Mendukung Browser Lama
+desc: menambahkan plugin legacy pada vite agar mendukung browser lama
```
Comment (elianiva) [inline content/membuat-vite-agar-mendukung-browser-lama.md:3]:
```suggestion
desc: menambahkan plugin legacy pada vite agar hasil build mendukung browser lama
```

File: content/membuat-vite-agar-mendukung-browser-lama.md [hunk 6]
```diff
@@ -0,0 +1,67 @@
+---
+title: Membuat Vite Agar Mendukung Browser Lama
```
Comment (elianiva) [inline content/membuat-vite-agar-mendukung-browser-lama.md:2]:
```suggestion
title: Membuat Website Yang Dibangun Menggunakan Vite Agar Mendukung Browser Lama
```

**Review by elianiva — CHANGES_REQUESTED** [review]
jangan lupa ganti huruf depan tiap paragraf jadi kapital

**Comment by bulukucing** [comment]
thanks om, sudah saya benarkan

**Review by aldy505 — APPROVED** [review]

**Review by elianiva — APPROVED** [review]

**Pull request merged**